### PR TITLE
feature/audio-creation-return-tuple

### PR DESCRIPTION
**Pull request by isaacna**
**Pull request recommendations:**
- [x] Name your pull request _your-development-type/short-description_. Ex: _feature/read-tiff-files_
- [ ] Link to any relevant issue in the PR description. Ex: _Resolves [gh-12], adds tiff file format support_
- [x] Provide context of changes.
 - Returning the the ffmpeg stdout and stderr files in addition to the audio path as a tuple
- [x] Provide relevant tests for your feature or bug fix.
- [ ] Provide or update documentation for any feature added by your pull request.

Thanks for contributing!

### CODE REVIEW

File: cdp_backend/tests/utils/test_file_utils.py [hunk 1]
```diff
@@ -89,14 +89,25 @@ def test_external_resource_copy(tmpdir: LocalPath, mocked_request: Generator) ->
...
     tmp_dir_audio_save_path = Path(tmpdir) / Path(audio_save_path).resolve()
 
     # Mock split
     with mock.patch("ffmpeg.run") as mocked_ffmpeg:
         mocked_ffmpeg.return_value = (b"OUTPUT", b"ERROR")
-        file_utils.split_audio(
-            video_read_path=example_video, audio_save_path=str(tmp_dir_audio_save_path)
-        )
+        try:
+            output = file_utils.split_audio(
```
Comment (JacksonMaxfield) [inline cdp_backend/tests/utils/test_file_utils.py:103]: You can unpack these as:

```python
audio_file, stdout_log, stderr_log = file_utils.split_audio(...)
```

File: cdp_backend/utils/file_utils.py [hunk 2]
```diff
@@ -113,8 +113,9 @@ def split_audio(
         Path to where the audio should be stored.
     Returns
     -------
-    audio_save_path: str
-        Path to where the split audio was saved.
+    save_paths: Tuple[str, str, str]
+        Tuple of path to where the split audio was saved, ffmpeg stdout path, and
+        ffmpeg stderr path.
```
Comment (JacksonMaxfield) [inline cdp_backend/utils/file_utils.py:118]: Make this multiple returns instead of a single Tuple? Pedantic I know. I'm sorry.

**Review by JacksonMaxfield — APPROVED** [review]
Minor changes but looks good to me.

File: cdp_backend/utils/file_utils.py (hunk 2 above)
Comment (isaacna) [reply cdp_backend/utils/file_utils.py]: Will do, I think it'll make it more clear

**Pull request merged**